stories/autocomplete.stories.ts

```diff
@@ -0,0 +1,63 @@
+import { storiesOf } from '@storybook/html';
+import { withHits } from '../.storybook/decorators';
+
+storiesOf('Autocomplete', module).add(
+  'default',
+  withHits(({ search, container, instantsearch }) => {
+    const instantSearchRatingAscSearchBox = document.createElement('div');
+    const instantSearchAutocomplete = document.createElement('div');
+
+    container.appendChild(instantSearchRatingAscSearchBox);
+    container.appendChild(instantSearchAutocomplete);
+
+    const customAutocomplete = instantsearch.connectors.connectAutocomplete(
+      renderOptions => {
+        const { indices, widgetParams } = renderOptions;
+
+        widgetParams.container.innerHTML = indices
+          .map(
+            ({ index, hits }) => `
+<li>
+  Index: <strong>${index}</strong>
+  <ol>
+    ${hits
+      .map(
+        hit => `<li>${instantsearch.highlight({ attribute: 'name', hit })}</li>`
+      )
+      .join('')}
+  </ol>
+</li>
+`
+          )
+          .join('');
+      }
+    );
+
+    search.addWidgets([
+      instantsearch.widgets
+        .index({ indexName: 'instant_search_price_asc' })
+        .addWidgets([
+          instantsearch.widgets.configure({
+            hitsPerPage: 3,
+          }),
+
+          instantsearch.widgets.searchBox({
```

Comment:
We don't really showcase the full power of the connector. IMO we should use the `refine` behaviour of the connector, rather than relying on the `searchBox` (for the design we can give it the same class).

Reply:
What could we showcase more with the search box connector that we don't with the widget?

Reply:
The story do not use `refine`, `currentRefinement` to implement the autocomplete. We leverage the `searchBox` widget. Most of the time people will build their autocomplete with the connector only, specially if it's an integration with a pre-built component. IMO it worth showcasing the full API.